gateway/rpc_storage_handler.go

```diff
@@ -817,13 +820,67 @@ func (gw *Gateway) getSessionAndCreate(keyName string, r *RPCStorageHandler, isH
 	}
 }
 
+func (gw *Gateway) ProcessOauthClientsOps(clients map[string]string) {
```

Comment:
It looks good! can we add a func description?

Reply:
done